### DISCUSSION TITLE
Prevent dropdown close when use Multi Select and Portal

### DISCUSSION THREAD

**Discussion by alexvarelaAA** [csandman/chakra-react-select#272] [Ideas]
` <Select
            tagVariant='solid'
            menuPortalTarget={document.getElementById('portal')}
            isMulti
            ... />`
            
I want to prevent the dropdown closes but I want to use a portal

**Comment by csandman** [comment]
If you're just asking about preventing the menu from closing when you select an option from it, passing `closeMenuOnSelect={false}` should do the trick? I'm not sure what this has to do with the portal specifically: https://codesandbox.io/s/kkkhz2?file=/app.tsx
